Formula/go.rb

```diff
@@ -47,11 +37,11 @@ class Go < Formula
   depends_on :macos => :mountain_lion
 
   # Should use the last stable binary release to bootstrap.
-  # More explicitly, leave this at 1.7 when 1.7.1 is released.
+  # More explicitly, leave this at 1.8 when 1.8.1 is released.
```

Comment:
I think this argument makes more sense to me than https://github.com/Homebrew/homebrew-core/pull/10048#discussion_r101629955. Fixed!